src/TableNew/BulkActions.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import {
+  Button, ButtonGroup, DropdownButton, Dropdown, useWindowSize,
+} from '..';
+import { SMALL_SCREEN_BREAKPOINT } from './constants';
+
+export const DROPDOWN_BUTTON_TEXT = 'More actions';
+export const SMALL_SCREEN_DROPDOWN_BUTTON_TEXT = 'Actions';
+
+const BulkActions = ({
+  actions, selectedRows, className, ...rest
+}) => {
+  const { width } = useWindowSize();
+  const [visibleActions, dropdownActions] = useMemo(() => {
+    const buttonActions = [...actions];
+    if (width < SMALL_SCREEN_BREAKPOINT) {
+      return [[], buttonActions];
+    }
+    let firstTwoActions = [];
+    let extraActions = [];
+    if (buttonActions.length <= 2) {
+      firstTwoActions = buttonActions;
+    } else {
+      firstTwoActions = buttonActions.splice(0, 2);
+      extraActions = buttonActions;
+    }
+    /*  Reversing the array because to the user it makes sense to put the primary button first,
+        but we want it on the right */
+    return [firstTwoActions.reverse(), extraActions];
+  }, [actions, width]);
+
+  return (
+    <div className={`pgn__bulk-actions ${className}`} {...rest}>
+      <ButtonGroup>
```

Comment:
Per the Figma mocks, it doesn't look the bulk actions should be in a `ButtonGroup`, but rather standalone buttons that are spaced.

![image](https://user-images.githubusercontent.com/2828721/104213831-b9476b80-5404-11eb-8648-18e21979f0bc.png)

Reply:
Yeah, just fixed that. Well, fixed the spacing. At a later date, @abutterworth is going to change the spacing on the ButtonGroup for our theme.
There were other reasons why ButtonGroup made sense but tbh I don't remember them right now.